### PR TITLE
Feature/members see all reviews

### PR DESCRIPTION
**Pull request by miggol**
Implements #93 and additionally allows committee members to view the details page of all reviews in the chamber they are a member of.

### CODE REVIEW

File: reviews/views.py [hunk 1]
```diff
@@ -327,12 +339,20 @@ def get_queryset(self):
...
+                       UsersOrGroupsAllowedMixin, generic.DetailView):
     """
     Shows the Decisions for a Review
     """
     model = Review
+    
+    def get_group_required(self):
+        
+        obj = self.get_object()
+        allowed_groups = [ settings.GROUP_SECRETARY ]
```
Comment (tymees) [inline reviews/views.py:352]: Kan dit niet beter een oneliner zijn?

``return [ settings.GROUP_SECRETARY, obj.proposal.reviewing_commitee.name ]``

Comment (miggol) [reply reviews/views.py]: Ja, en ik even de variabelenaam consistent gemaakt met de soortgelijke functies. 

Soms vind ik de eerste stijl wel lekker idiomatisch, dat je in íeder geval de secretaris toelaat en daar nog iets aan toevoegt. Maar met maar twee stappen is dat een beetje lelijk.

**Comment by miggol** [comment]
Before merging I want to double-check if it's safe to assume that proposal.reviewing_commitee.name will always reference a group. I think we do that elsewhere too but maybe there's a better way I don't know of.

**Comment by tymees** [comment]
A doublecheck is always good, but I think in this case it should always be the case in production.

This attribute is set on the very first page of the form, and is hardcoded against the default groups. The only reason I see that this won't happen is:
1. The groups aren't present at all (in which case we fucked up royally anyway)
2. A proposal was created outside the form (in which case we probably fucked up royally again)

**Pull request merged**